apps/climatemappedafrica/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "climatemappedafrica",
-  "version": "0.0.2",
+  "version": "0.0.3",
```

Comment:
The latest image in dockerhub is at 0.0.3 already. Plus the branch won't build the ARM image since you do not have that functionality in your branch/PR. You can make the env change I will pull and make the version change if need be.

Reply:
@VinneyJ I do not see version 3 in dockerhub. This is also to ensure pushing to dokku will work.

Reply:
Awesome then. We can merge.